Peer dependency warning with immutable@4.0.0-rc.12

It seems like the semver specification for immutable in the peerDependencies section doesn't work right for v4.

My package.json has the latest immutable version:
```
  "dependencies": {
    ...
    "immutable": "^4.0.0-rc.12",
    ...
  },
```

Result:
`npm ERR! peer dep missing: immutable@^3||^4, required by @americanexpress/vitruvius@2.0.0`

Maybe ^4 does not match the pre-release versions?

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "rimraf": "^3.0.0"
   },
   "peerDependencies": {
-    "immutable": "^3||^4",
+    "immutable": "^3||^4.0.0-rc",
     "redux": "^3||^4",
     "redux-immutable": "^4.0.0"
   },

